integrations/astradb.md

```diff
@@ -55,32 +55,19 @@ pip install astra-haystack
 This package includes Astra Document Store and Astra Retriever classes that integrate with Haystack 2.0, allowing you to easily perform document retrieval or RAG with AstraDB, and include those functions in Haystack pipelines.
 
 In order to connect AstraDB with Haystack, you'll need these pieces of information from your Datastax console:
-- AstraDB ID
-- region
+- API Endpoint
+- Token
 - Astra collection name
 - Astra keyspace name
-- access token
 
 ### how to use the `AstraDocumentStore`:
 
 ```python
 from haystack import Document
 from haystack_integrations.document_stores.astra import AstraDocumentStore
 
-
-astra_id = os.getenv("ASTRA_DB_ID", "")
-astra_region = os.getenv("ASTRA_DB_REGION", "us-east-2")
-astra_application_token = os.getenv("ASTRA_DB_APPLICATION_TOKEN", "")
-collection_name = os.getenv("COLLECTION_NAME", "haystack_integration")
-keyspace_name = os.getenv("KEYSPACE_NAME", "astra_haystack_test")
-
-document_store = AstraDocumentStore(
-    astra_id=astra_id,
-    astra_region=astra_region,
-    astra_collection=collection_name,
-    astra_keyspace=keyspace_name,
-    astra_application_token=astra_application_token,
-)
+# Make sure ASTRA_API_ENDPOINT and ASTRA_TOKEN environment variables are set
```

Comment:
How about `Astra collection name` and `Astra keyspace name`, don't we need them anymore? They are still in the list above alongside `API Endpoint` and `Token`

Reply:
I added default values for them, so I'd say in this context no, we don't need them anymore.